data-reporting-service/person-service/src/test/java/gov/cdc/etldatapipeline/person/config/PersonSvcYmlPropertiesTest.java

```diff
@@ -18,16 +18,19 @@ public class PersonSvcYmlPropertiesTest {
 
     @Autowired
     private KafkaConfig kafkaConfig;
+
     @Test
     void whenBindingYMLConfigFile_thenAllFieldsAreSet() {
         Assertions.assertEquals("cdc.nbs_odse.dbo.Default.input", kafkaConfig.getDefaultDataTopicName());
         Assertions.assertEquals("cdc.nbs_odse.dbo.Person", kafkaConfig.getPersonTopicName());
```

Comment:
qq, should we update the tests to get the topics wrt the naming convention (eg nbs_Person)?

Reply:
Sure will do.

Reply:
These are the incoming topics from debezium. I guess they are produced in the above format. 
The output topics follow the naming convention: `nbs_patient`

Reply:
Quick update: The input topics from debezium should be following the naming convention [nbs_Person](url) but I don't know if that should be taken up in another pr or not. I can switch the naming convention for testing.